mail-mta/opensmtpd/Manifest

```diff
@@ -1 +1,3 @@
 DIST opensmtpd-7.3.0p1.tar.gz 849026 BLAKE2B bdb83468df12af2a2b39664a4408acfaf0cd81551074fb3823f7eb1b0d30ce79d812876fcea2ff940ddaac8b1a96700e1f4d2c275beab106fed010b733cff5c2 SHA512 2106de43e4b7435e49df759570ec758672ca8271dc451e30a261c250b41908b6ffe28e571a5f52e2ac14f59af132d1df45b272f8fcafeab04fcfb6bd5db970bb
+DIST opensmtpd-7.4.0p1.sum.sig 256 BLAKE2B 7d2e112242578a049e7bf597a6eaf240bd5a4f6dc68c9401071351d6cf8898e04e601454f0151bbb14b5a522eb3c90601e3b2b1be882b749995541a3660bf176 SHA512 78e8b4ea5d464fc1699a27f69ea12d9831efe795ef0128c1a0ab8ccf8279d0ec9e28db3f028e024371445405af2a09e7132e4492b4c23ae859f868650b3aabce
+DIST opensmtpd-7.4.0p1.tar.gz 907582 BLAKE2B 5da835028a4f7e463d9b1f9f310086c1c3ff226fba15617eee6fb1f67b718f0c9c5424cb5702a19e0ac8990af39a8cbfbac7dfc2dd3fd2df2a49c980e410a9d5 SHA512 5bce92d938ddb7fe327c1e8a1094b9b376489784932999ad8bb827c94efc7a29874e2dea1ed0a183ae9486ca4c9108a7381aa98052a3cb1e274a53d18b6c5cea
```

Comment:
I had a mismatch with this, had to regenerate the manifest.

Reply:
1. I inherited `verify-sig` in a `sec-keys` package and of course it failed.
2. Looks like tarballs were replaced at some moment. Signature verification passes so it's ok to regenerate hashes.

Both fixed